src/main/java/io/jenkins/plugins/oak9/Oak9Builder.java

```diff
@@ -177,6 +190,17 @@ public int getMaxSeverity() {
         return this.maxSeverity;
     }
 
+    /**
+     * The amount of time to wait for a validation to complete
+     * @return the polling timeout
+     */
+    public int getPollingTimeoutSeconds() {
+        if (this.pollingTimeoutSeconds <= 0) {
+            return 30;
```

Comment:
In the spirit of removing the hard-coding, should this `30` default be a setting/constant somewhere?